### PR TITLE
Create CONTRIBUTING.md [ci skip]

### PR DESCRIPTION
**Pull request by sahbi-ktifa**
-   Please make sure the below checklist is followed for Pull Requests.

-   [ ] [Travis tests](https://travis-ci.org/jhipster/jhipster-vuejs/pull_requests) are green
-   [ ] Tests are added where necessary
-   [ ] Documentation is added/updated where necessary
-   [ ] Coding Rules & Commit Guidelines as per our [CONTRIBUTING.md document](https://github.com/jhipster/jhipster-vuejs/blob/master/CONTRIBUTING.md) are followed

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,349 @@
...
+    git pull --ff upstream master
+    ```
+
+## <a name="setup"></a> Generator development setup
+
+JHipster is a [Yeoman Generator](http://yeoman.io/), so you must follow the [Yeoman authoring documentation](http://yeoman.io/authoring/) in order to be able to run and test your changes.
+
+Here are the most important steps.
+
+### Fork the generator-jhipster-vuejs project
```
Comment (pascalgrimaud) [inline CONTRIBUTING.md]: `jhipster-vuejs` instead of `generator-jhipster-vuejs`

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,349 @@
...
+
+Go to the [generator-jhipster-vuejs project](https://github.com/jhipster/generator-jhipster-vuejs) and click on the "fork" button. You can then clone your own fork of the project, and start working on it.
```
Comment (pascalgrimaud) [inline CONTRIBUTING.md]: same comment. Update the link

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,349 @@
...
+
+[Please read the GitHub forking documentation for more information](https://help.github.com/articles/fork-a-repo)
+
+### Set NPM/YARN to use the cloned project
+
+In your cloned `generator-jhipster-vuejs` project, type `npm link` or `yarn && yarn link` depending on the package manager you use.
```
Comment (pascalgrimaud) [inline CONTRIBUTING.md]: same

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,349 @@
...
+
+This will do a symbolic link from the global `node_modules` version to point to this folder, so when we run `jhipster --blueprint vuejs`, you will now use the development version of JHipster.
+
+For testing, you will want to generate an application, and there is a specific issue here: for each application, JHipster installs a local version of itself. This is made to enable several applications to each use a specific JHipster version (application A uses JHipster 3.1.0, and application B uses JHipster 3.2.0).
+
+To overcome this you need to run `npm link generator-jhipster-vuejs` or `yarn link generator-jhipster-vuejs` on the generated project folder as well, so that the local version has a symbolic link to the development version of JHipster.
```
Comment (pascalgrimaud) [inline CONTRIBUTING.md]: same

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -0,0 +1,349 @@
...
+
+To put it in a nutshell, you need to:
+
+1.  run `npm link` or `yarn link` on the `generator-jhipster-vuejs` project
```
Comment (pascalgrimaud) [inline CONTRIBUTING.md]: same

File: CONTRIBUTING.md [hunk 6]
```diff
@@ -0,0 +1,349 @@
...
+2.  run `npm link generator-jhipster-vuejs` or `yarn link generator-jhipster-vuejs` on the generated application folder (you need to do this for each application you create)
```
Comment (pascalgrimaud) [inline CONTRIBUTING.md]: same

File: CONTRIBUTING.md [hunk 7]
```diff
@@ -0,0 +1,349 @@
...
+
+## <a name="question"></a> Questions and help
+
+This is the JHipster bug tracker, and it is used for [Issues and Bugs](#issue) and for [Feature Requests](#feature). It is **not** a help desk or a support forum.
+
+If you have a question on using JHipster, or if you need help with your JHipster project, please [read our help page](https://www.jhipster.tech/help/) and use the [JHipster tag on StackOverflow](http://stackoverflow.com/tags/jhipster) or join our [Gitter.im chat room](https://gitter.im/jhipster/generator-jhipster).
+
+## <a name="issue"></a> Issues and Bugs
+
+If you find a bug in the source code or a mistake in the documentation, you can help us by [submitting a ticket](https://opensource.guide/how-to-contribute/#opening-an-issue) to our [GitHub issues](https://github.com/jhipster/jhipster-vuejs/issues). Even better, you can submit a Pull Request to our [JHipster generator project](https://github.com/jhipster/generator-jhipster) or to our [Documentation project](https://github.com/jhipster/jhipster.github.io).
```
Comment (pascalgrimaud) [inline CONTRIBUTING.md]: same

**Comment by sahbi-ktifa** [comment]
Changes made, could you review it again @pascalgrimaud plz ?

**Pull request merged**